client/my-sites/exporter/post-type-options.jsx

```diff
@@ -135,7 +81,25 @@ const PostTypeOptions = React.createClass( {
 					</p>
 				}
 
-				{ this.renderFields() }
+				<div className="exporter__option-fieldset-fields">
+					{ fields.map( fieldName =>
+						<Select key={ fieldName }
+							ref={ fieldName }
```

Comment:
String refs are legacy. Let's use callbacks: https://facebook.github.io/react/docs/more-about-refs.html#the-ref-callback-attribute